Use printf instead of echo -n to avoid literal "-n" printed onscreen.
--- configure.orig	2011-07-31 19:31:11.000000000 -0500
+++ configure	2011-09-12 21:47:50.000000000 -0500
@@ -16936,7 +16936,7 @@
 echo "   Datadir:       $datadir"
 echo "   Mandir:        $mandir"
 echo "   CFLAGS:        $CFLAGS"
-echo -n "   Sound:         "
+printf "   Sound:         "
 
    if test "x$MIKMOD" = "x1"; then
       echo "Yes"
@@ -16944,7 +16944,7 @@
       echo "NO"
    fi
 
-echo -n "   Network:       "
+printf "   Network:       "
 
    if test "x$NETWORK" = "x1"; then
       echo "Yes"
@@ -16952,7 +16952,7 @@
       echo "NO"
    fi
 
-echo -n "   Readline:      "
+printf "   Readline:      "
 
    if test "x$READLINE" = "x1"; then
       echo "Yes"
@@ -16960,7 +16960,7 @@
       echo "NO"
    fi
 
-echo -n "   GTK 2+:        "
+printf "   GTK 2+:        "
 
    if test "x$GTK20" = "x1"; then
       echo "Yes"
